### PR TITLE
Force HTTPS

### LINKED ISSUE #125: Enforce HTTPS usage

### PR DESCRIPTION
**Pull request by samuelli**
Fixes #125

### CODE REVIEW

File: src/server/dash-server.ts [hunk 1]
```diff
@@ -76,6 +77,15 @@ export class DashServer {
     }
   }
 
+  enforceHTTPS(req: express.Request, res: express.Response, next: express.NextFunction) {
+    if (!req.secure && process.env.NODE_ENV === 'production') {
+      res.header('Location', `https://${req.headers['host']}${req.url}`);
```
Comment (aomarks) [inline src/server/dash-server.ts]: you can use `res.redirect(code, url)`

File: src/server/dash-server.ts [hunk 2]
```diff
@@ -76,6 +77,15 @@ export class DashServer {
...
+      res.sendStatus(301);
+      return;
```
Comment (aomarks) [inline src/server/dash-server.ts]: use else instead of return

File: src/server/dash-server.ts (hunk 1 above)
Comment (samuelli) [reply src/server/dash-server.ts]: Oh cool, thanks.

File: src/server/dash-server.ts (hunk 2 above)
Comment (samuelli) [reply src/server/dash-server.ts]: Done.

**Pull request merged**